public/app/plugins/datasource/cloudwatch/utils/query/getStatsGroups.ts

```diff
@@ -0,0 +1,38 @@
+const antlr4 = require('antlr4');
+const ScrollQLLexer = require('./ScrollQLLexer').ScrollQLLexer;
+const ScrollQLParser = require('./ScrollQLParser').ScrollQLParser;
+const ScrollQLParserListener = require('./ScrollQLParserListener').ScrollQLParserListener;
+
+class GroupListener extends ScrollQLParserListener {
+  groupNames: string[] = [];
+
+  enterLogStats(ctx: any) {
```

Comment:
Can we add the ctx type here?

Reply:
Tried using the ctx type as defined in `@types/antlr4` but it doesn't seem to match the actual structure of the object, so it wasn't very useful. I could create an interface definition though